libraries/AP_InertialSensor/AP_InertialSensor_BMI088.cpp

```diff
@@ -400,7 +400,7 @@ void AP_InertialSensor_BMI088::read_fifo_gyro(void)
     }
     const float scale = radians(2000.0f) / 32767.0f;
     const uint8_t max_frames = 8;
-    const Vector3i bad_frame{int16_t(0xffff), int16_t(0xffff), int16_t(0xffff)};
+    const Vector3i bad_frame{INT16_MIN,INT16_MIN,INT16_MIN};
```

Comment:
These are not -32767 - please fix code or PR comments....

Reply:
fixed commit comment